python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/queue_monitor_length.schema.yml

```
@@ -55,3 +55,56 @@ keys:
       tx_latency:
         type: bool
         description: "Enable tx-latency mode."
+      mirror:
+        description: Enable frame mirroring during congestion.
+        type: dict
+        keys:
+          enabled:
+            type: bool
+          destination:
+            description: Mirror destination.
+            type: dict
+            keys:
+              cpu:
+                type: bool
+                description: CPU ports.
+              ethernet_interfaces:
+                type: list
+                items:
+                  type: str
+                  description: |-
+                    Hardware Ethernet interface name.
+                    Example - Ethernet4
+              tunnel_mode_gre:
+                type: dict
+                keys:
+                  source:
+                    type: str
+                    required: true
+                    description: Source IP address of GRE tunnel.
+                  destination:
+                    type: str
+                    required: true
+                    description: Destination IP address of GRE tunnel.
+                  dscp:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 0
+                    max: 63
+                    description: DSCP of the GRE tunnel. EOS default is 0.
+                  ttl:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 255
+                    description: TTL range. EOS default is 128.
+                  protocol:
+                    type: str
+                    description: |-
+                      Protocol type in GRE header. Protocol range - 0x0000-0xFFFF.
+                      EOS default is 35006.
```

This is confusing when the range is 0x0000 - 0xFFFF. You should write the default here as the hex value.
```suggestion
                      EOS default is 0x88BE.
```